stress-test/user-files/simulations/rinhabackend/RinhaBackendSimulation.scala

```diff
@@ -23,7 +23,12 @@ class RinhaBackendSimulation
       // 422 pra requests inválidos :|
       // 400 pra requests bosta tipo data errada, tipos errados, etc. :(
       .check(status.in(201, 422, 400))
-      .check(header("Location").optional.saveAs("location"))
+      // Se a criacao foi na api1 e esse location request atingir api2, a api2 tem que encontrar o registro.
+      // Pode ser que o request atinga a mesma instancia, mas estatisticamente, pelo menos um request vai atingir a outra.
+      // Isso garante o teste de consistencia de dados
+      .checkIf(session => session("httpStatus").as[String] == "201") {
+        header("Location").saveAs("location")
+      }
```

Comment:
essa modificação garante apenas que o header location será salvo se o status code for 201, ou seja previne alguém de retornar esquisitamente um header location para os status 422 ou 400 🤷🏻

Reply:
Mas se o status for 201, ele vai obrigatoriamente checar a presença do header Location e armazenar a URL.

Logo depois, ele vai executar a consulta, e é nesse ponto que 100% das chamadas precisam ser 200.